src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -156,6 +209,8 @@ const useFetchChannelHook = () => {
     try {
       await saveAllChannelData(anonymousChannel, 'ANONYMOUS');
       refresh('channelList');
+      const timestamp = new Date().toISOString().split('T')[0];
+      StorageUtils.channelAnonTimeStamps.set(timestamp);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the addition of a timestamp to track when the channels were last fetched is a good practice for caching data. Again, it would be better if there was some error handling around the `StorageUtils.channelAnonTimeStamps.set(timestamp);` line in case there are any issues with setting the timestamp in storage.


<!-- This is an auto-generated comment by OSS CodeRabbit -->